src/app/components/containers/settings-container/settings/settings.component.html

```diff
@@ -452,3 +452,14 @@
     <option [value]="ResearchSpeed.Speed6">Research Speed 6</option>
   </select>
 </div>
+<div class="label">Theme</div>
+<div class="field">
+  <select
+    title="Choose Factorio Lab theme"
+    [ngModel]="settings.theme"
+    (change)="emitAny(setTheme, $event)"
```

Comment:
I think you can just use the existing `emitNumber` function for this, as Research Speed dropdown does. Currently this branch shows missing coverage for the `emitAny` function, reusing the `emitNumber` would resolve that issue. 
Sure, it does an unnecessary `Math.round` call, but it shouldn't hurt anything.

Reply:
Theme is a string though?

I'll write a test for emitAny.

Reply:
Duhhh. I've been thinking of it as a numerical enum. New test will be good.